torchvision/tv_tensors/__init__.py

```diff
@@ -1,6 +1,6 @@
 import torch
 
-from ._bounding_boxes import BoundingBoxes, BoundingBoxFormat, is_rotated_bounding_format
+from ._bounding_boxes import BoundingBoxes, BoundingBoxFormat, CLAMPING_MODE_TYPE, is_rotated_bounding_format
```

Comment:
With `CLAMPING_MODE_TYPE` now being used for both bounding boxes and keypoints alike, it would make sense to move its definition out of `_bounding_boxes`, e.g., directly to `tv_tensors.__init__`.